src/api/app/assets/stylesheets/webui/responsive_ux.scss

```diff
@@ -73,7 +73,19 @@ body.responsive-ux {
     #notifications-filter-desktop {
       &.show { border-top: 1px solid $gray-300; }
       &.sticky-top { top: $top-bar-height; }
-      #filters { height: 200px; overflow: auto; }
+      #filters {
+        max-height: 100vw; overflow: auto;
```

Comment:
For smaller mobile screens, the filters are now fully taking the vertical space of the viewport. This prevent users from scrolling the page when filters are open. I guess this isn't really an issue, just wanted to be sure that you're aware of this.